nix/overlay.nix

```diff
@@ -119,6 +119,11 @@ rec {
       };
   };
 
+  iree-runtime = final.callPackage ./pkgs/iree-runtime.nix {
+    inherit rv32-stdenv;
+    inherit iree;
```

Comment:
These attribute inheritance might not required as `final.callPackage` will automatically do dependencies injection.